### ISSUE TITLE
Create a module skin repository

### ISSUE THREAD

**Issue by alexrighetto** [stubbornella/oocss#32]
It could be a nice idea  creating a module skin repository (o a marketplace), feeded by community, in order to share them easily, based on Stubbornella's module structure.
the pluspoints are:
1 Share the tecnology
2 simplify the develop
3 a place where the designer can show his skills 

thanks

**Comment by stubbornella** [comment]
A fantastic idea... any chance you are interested in building it?

**Comment by alexrighetto** [comment]
Yes I'm proud to have a chance to build it! Any suggestions or guidelines to start?
Wow, Thanks again.

**Comment by martinklepsch** [comment]
In fact this sounds like a fun idea.  
Create a set of modular markups and let people style them with CSS & JS.
Maybe with some live editors as they are around.

Other people can then give points or 'like' your 'art'.  
Other Features could include:   
- Rebounds like in Dribbble  
- Some kind of Badges for "Fast", "Pretty", "blah"  

This could be really interesting.

**Comment by alexrighetto** [comment]
Exactly, these are good ideas.

The home page should show the top rated skins.
